tests/test_tdigest.py

```diff
@@ -130,6 +130,17 @@ def test_adding_centroid_with_exisiting_key_does_not_break_synchronicity(self, e
         td._add_centroid(Centroid(-1.1, 10))
         assert all([k == centroid.mean for k, centroid in td.C.items()])
 
+    def test_quantile_with_single_centroid(self, empty_tdigest):
+        td = empty_tdigest
+        td.update(1)
+        assert td.quantile(1) == 0.5
+
+    def test_quantile_with_single_centroid_at_zero(self, empty_tdigest):
+        td = empty_tdigest
+        td.update(0)
+        assert td.quantile(0) == 0.5
```

Comment:
should this be 0.5? The quantile is the value `p` s.t. `F(q) = p`. If `F` is equal to 0 before 0, and 1 at and after 0, then I would expect this to be 1. (That's kind of confusing because of all the 0 and 1...)